### ISSUE TITLE
Cleaner Menu Bar (Hide Unshaky Completely)

### ISSUE THREAD

**Issue by ahmed-alii** [aahung/Unshaky#184]
Unshaky works the best and its saving my hundreds $$ for some days now.
However, having it hidden from the menu bar helps a lot for somebody with **OCD** like me.

Can we make this happen?

**Comment by 0xPhong** [comment]
Yeah. This application like a charm and will be more shine if got an option to hide icon on menubar.

**Comment by ThijmenDam** [comment]
A third party solution: [BarTender](https://www.macbartender.com)

NB: I do not have any affiliation with BarTender.

**Comment by phongdotvn** [comment]
> A third party solution: [BarTender](https://www.macbartender.com)
> 
> 
> 
> NB: I do not have any affiliation with BarTender.

Or using this 6.6k stars opensource for Hidden Bar

https://github.com/dwarvesf/hidden

**Comment by ThijmenDam** [comment]
> https://github.com/dwarvesf/hidden

I had absolutely no idea this project existed. Thank you very much!

**Comment by rpendleton** [comment]
This is actually possible already. It's documented in [How to hide Unshaky from status bar](https://github.com/aahung/Unshaky/wiki/How-to-hide-Unshaky-from-status-bar), but basically, you just have to hold command while dragging the icon away from the menu bar.

**Comment by Bluekezhou** [comment]
您的邮件已收到，谢谢！
